static/src/javascripts/projects/common/views/weather.html

```diff
@@ -1,28 +1,30 @@
 <div class="weather">
-    <ul class="u-unstyled weather__forecasts">
-        <li class="weather__forecast">
-            <p class="weather__desc">
-                <span class="u-h">The temperature</span>
-                <span class="weather__time">now</span>
-                <span class="u-h">is</span>
-                <span class="weather__temp js-weather-temp">{{tempNow}}</span>
-            </p>
-            <i class="i i-weather-{{icon}} weather__icon js-weather-icon" title="{{description}}"><span class="u-h">{{description}}</span></i>
-        </li>
-    </ul>
+    <div class="u-unstyled weather__current">
+        <p class="weather__desc">
+            <span class="u-h">The temperature</span>
+            <span class="weather__time">now</span>
+            <span class="u-h">is</span>
+            <span class="weather__temp js-weather-temp">{{tempNow}}</span>
+        </p>
+        <i class="i i-weather-{{icon}} weather__icon js-weather-icon" title="{{description}}"><span class="u-h">{{description}}</span></i>
+    </div>
+    <a class="weather__toggle-forecast js-toggle-forecast" data-link-name="weather-toggle-forecast" data-toggle-class="js-weather-forecasts-mobile"
+        href="#toggle-forecast"><span class="u-h">Toggle forecast</span><i class="i i-arrow-grey-large weather__toggle-icon"></i>
+    </a>
```

Comment:
This should not be an anchor. Please use a `<button>` instead, and consider using some nice accessible ARIA attributes, whilst you are at it.